src/awesomeSezzle.js

```diff
@@ -675,25 +675,30 @@ class AwesomeSezzle {
 		return currency || 36;
 	}
 
-	calculateInterest(price, term, APR) {
+	calculateMonthlyWithInterest(price, term, APR) {
 		if(APR > 0){
 			const rate = (APR/100)/12;
 			const numerator = price * rate * Math.pow(1+rate, term);
 			const denominator = Math.pow(1+rate, term)-1;
 			const interestPayment = numerator/denominator
 			return interestPayment;
 		} else {
-			return price/term;
+			return (price/term).toFixed(2);
 		}
 	}
-	calculateMonthly(priceString, parseMode, term, APR) {
-		const interestAmount = this.calculateInterest(priceString, term, APR);
-		return this.addDelimiters(interestAmount.toFixed(2), parseMode);
+	formatMonthly(priceString, parseMode, term, APR) {
```

Comment:
`calculateMonthlyWithInterest` expects a float number as an input for the `price`, but we send a string

Reply:
It only ever multiplies, so it doesn't care if it receives a number or a string - does it matter? If so, for the sake of dryness, can we open that function with converting it to number instead of everywhere it is called?

Reply:
Yeah, converting it to float in the beginning of the function works. Even though for the widget price case it will be unnecessary because it already sends the float. However, as I mentioned in the general comment, I am not sure why we need to calculate monthly payment 3 times for each term.

About whether it matters: as long as the `priceString` is always guaranteed to be convertible to a float, the multiplication will convert it with no problem, so it's not an issue of a buggy behaviour. However, it makes code more difficult to follow and in general, for me coming from a strongly-typed language background, it feels dangerous and bizarre :)